Python/alerts-to-discord/main.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pprint
+
+# [START v2import]
+from firebase_functions import params
+from firebase_functions.alerts import (
+    app_distribution_fn,
+    crashlytics_fn,
+    performance_fn,
+)
+# [END v2import]
+
+import requests
+
+DISCORD_WEBHOOK_URL = params.SecretParam("DISCORD_WEBHOOK_URL")
+
+
+def post_message_to_discord(bot_name: str,
```

Comment:
need to add `secrets: [DISCORD_WEBOOK_URL]` here so that the secret is bound to the function

Reply:
oops, this is a helper. Might be safer to pass the webhook URL in so that any caller knows it needs the secret to be bound, but not a blocker